guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>The <code>arcgis.env</code> module provides a shared environment used by the different modules. It stores globals, such as the currently active GIS, the default geocoder, and more. &nbsp;It also stores environment settings that are common among all tools, such as the output spatial reference, cell size, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Missing comma and space edits</p>